rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSet.java

```diff
@@ -924,10 +954,10 @@
      */
     public static final CommandSet.Entry[] RM_ONLY = new CommandSet.Entry[] { LOGIN, NODE_ADD, NODE_LIST, NODE_INFO,
                                                                               NODE_LOCK, NODE_UNLOCK, NODE_REMOVE,
-                                                                              NS_CREATE, NS_LIST, NS_REMOVE, NS,
-                                                                              INFRASTRUCTURE, INFRASTRUCTURE_LIST,
-                                                                              POLICY, POLICY_LIST, TOPOLOGY, FORCE,
-                                                                              RM_STATS, RM_HELP };
+                                                                              NS_DEFINE, NS_CREATE, NS_DEPLOY, NS_LIST,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "NS_CREATE"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)